k8s/etcd/README.md

```diff
@@ -239,13 +239,13 @@ kubectl exec -it "${APP_INSTANCE_NAME}-etcd-0" --namespace "${NAMESPACE}" -- etc
 ```
 
 #### Interact with etcd service using port-forwarding
-To connect to etcd service from your machine, install latest release of etcd binaries. For installation of etcd binaries, visit [official etcd documentation](https://github.com/etcd-io/etcd/releases).
+To connect to etcd service from your machine, install the latest release of the [etcd binaries](https://github.com/etcd-io/etcd/releases).
```

Comment:
Please add a blank line after a header.

https://github.com/google/styleguide/blob/gh-pages/docguide/style.md#add-spacing-to-headings